packages/vue-router/tests/vue-router-routing-provider.test.ts

```diff
@@ -44,7 +45,7 @@ describe('VueRouterRoutingProvider', () => {
         '[FullPageLoad]/some-route',
         { }
       )
-      const startRouteChangeSpan = jest.fn(() => spanFactory.toPublicApi(span))
+      const startRouteChangeSpan: StartRouteChangeCallback = jest.fn(() => spanFactory.toPublicApi(span) as RouteChangeSpan)
```

Comment:
the span factory returns `Span` but start route change callback now returns `RouteChangeSpan` with the extended `end` signature